mmengine/runner/runner.py

```diff
@@ -1722,6 +1724,17 @@ def train(self) -> nn.Module:
         # initialize the model weights
         self._init_model_weights()
 
+        # try to enable gredient_checkpoint feature
+        modules = self.cfg.get('gredient_checkpoint', None)
+        if modules is not None:
+            self.logger.info(f'Enabling the "gredient_checkpoint" feature'
+                             f'for sub-modules: {modules}')
+            for module in modules:
+                assert hasattr(
+                    ori_model,
+                    module), (f'Your model does not have {module} module')
```

Comment:
```python
from operator using attrgetter
getter = attrgetter('module')
turn_on_gredient_checkpoint(getter(ori_model, module))
```
Support getting inner child module

Reply:
BTW, If users have specified the gradient checkpoint module, maybe should just checkpoint the module rather than checkpoint its child further.